src/context.h

```diff
@@ -9,8 +9,8 @@
 
 #define CTX_VIEWPORT ((Rectangle) { 0, 0, CTX_VIEWPORT_WIDTH, CTX_VIEWPORT_HEIGHT })
 
-#define DEFAULT_WINDOW_WIDTH (CTX_VIEWPORT_WIDTH * 3)
-#define DEFAULT_WINDOW_HEIGHT (CTX_VIEWPORT_HEIGHT * 3)
+#define DEFAULT_WINDOW_WIDTH (CTX_VIEWPORT_WIDTH * 4)
+#define DEFAULT_WINDOW_HEIGHT (CTX_VIEWPORT_HEIGHT * 4)
```

Comment:
I think this is a large part of why performance sucks on mobile; however, there is no easy way to address this without using JavaScript...